service/src/main/test/java/com/carhouse/service/impl/FuelTypeServiceImplTest.java

```diff
@@ -0,0 +1,73 @@
+package com.carhouse.service.impl;
+
+import com.carhouse.dao.impl.FuelTypeDaoImpl;
+import com.carhouse.model.FuelType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+
+@ExtendWith(MockitoExtension.class)
+class FuelTypeServiceImplTest {
+
+    @Mock
+    private FuelTypeDaoImpl fuelTypeDao;
+
+    @InjectMocks
+    private FuelTypeServiceImpl fuelTypeService;
+
+    private List<FuelType> listFuelTypes;
+    @BeforeEach
+    void addFuelTypesToService() {
+        listFuelTypes = new ArrayList<>(){{
```

Comment:
can be replaced to 
` private List<FuelType> listFuelTypes = new ArrayList<>(){{ ... }};`